src/components/Paging/Paging.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { Paging } from './Paging';
+import { Button } from '@rmwc/button';
+
+describe('Paging', () => {
+  it('should render all pages', () => {
+    const onPageChange = jest.fn(() => {});
+    const paging = shallow(
+      <Paging pages={10} pageToStartWith={1} onPageChange={onPageChange} />,
+    );
+
+    expect(paging.find(Button)).toHaveLength(10 + 2);
```

Comment:
Is there a better way to test this?

Reply:
if the buttons are exactly the same, no. But if the numbers for example have a id which is separating them from the arrows you can check that